src/lib/server/twilio.ts

```diff
@@ -263,13 +265,19 @@ export const getMsg = async (url: URL) => {
 	return response;
 };
 
+function shuffleArr(arr: any[]) {
+	for (let i = arr.length - 1; i > 0; i--) {
+		const j = Math.floor(Math.random() * (i + 1));
```

Comment:
If Math.random() ever returns exactly 1 (rare but possible), this will fail with out of bounds if it happens on the first iteration

Reply:
Nevermind, if this is the browser-compliant Math.random() it excludes one. Carry on